frontend/packages/process-editor/src/components/ConfigPanel/ConfigEndEvent/CustomReceiptContent/CustomReceipt/CustomReceipt.tsx

```diff
@@ -83,7 +79,7 @@ export const CustomReceipt = (): React.ReactElement => {
         />
         <EditDataType
```

Comment:
And I think we cannot use this component here anymore, since it also does a call to mutateDataType. A solution can be to add a callBack function in stead and use this current logic, that implements mutateDataType, only for the normal tasks. While for the custom receipt should we might need to fix a new endpoint in backend that only updates the dataType in the layoutset, or we can create a new hook that that first calls `deleteLayoutSet` and then adds a new layoutset again with the updated dataType. What do you think is the best solution?

Reply:
I think we have fixed this in backend now? 😄